### DISCUSSION TITLE
https://vlaaad.github.io/2019-03-30/question-marks-in-clojure

### DISCUSSION THREAD

**Discussion by giscus** [vlaaad/vlaaad.github.io#12] [Posts]
# https://vlaaad.github.io/2019-03-30/question-marks-in-clojure

Dos and don’ts about using question mark suffix for naming things in Clojure

https://vlaaad.github.io/2019-03-30/question-marks-in-clojure

**Comment by 1pedro** [comment]
I'm learning Clojure right now and I'm struggling with the question mark because Clojure doesn't seem to have a pattern.
For example, the `some` function doesn't have the question mark but the `every?` function has.. looks weird.

But make sense of what you've said!

**Reply by vlaaad** [reply]
I think this is because `every?` is a predicate (returns boolean), while `some` is a short-circuiting filter (returns value returned by filtering function)
